LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -1,4 +1,5 @@
 {
+    "IndexedDB": "import", 
```

Comment:
this is already below in this file (as web-platform-tests/IndexedDB).

Can you please undo this change?